integrations/facebook-pixel/test/index.test.js

```diff
@@ -1721,6 +1752,38 @@ describe('Facebook Pixel', function() {
         });
         assertEventId(window.fbq);
       });
+
+      it('should send facebook pixel properties for product added', function() {
+        analytics.track('Product Added', {
```

Comment:
"Product Added" events are formatted like this: https://segment.com/docs/spec/ecommerce/v2/#product-added

Reply:
Got that.